rusk/src/lib/services/state.rs

```diff
@@ -329,14 +326,29 @@ impl State for Rusk {
         info!("Received Revert request");
 
         let mut state = self.state()?;
-
         state.revert();
-        self.persist(&mut state)?;
```

Comment:
I think that we should keep the persist in the revert, node-side will always call the persist after each revert

Reply:
If the idea is to make persist a different call for flexibility, I recommend also doing so on `revert`, since anything else is sort of sneaky. Either the node controls persistence or it doesn't.